public/studies-schema.json

```diff
@@ -87,6 +88,9 @@
     },
     "order": {
       "type": "number"
+    },
+    "minimumCoreVersion": {
+      "type": "number"
```

Comment:
Is number the right type?  cause we're going to have things that are like `1.2.3` i would have thought the multiple `.` would require that we use string?

Reply:
Good catch, thanks!